src/data/data.ts

```diff
@@ -239,25 +240,46 @@ const noContacts = {
   contactsMultiplier: 0,
 }
 
-export const RiskProfile: { [key: string]: PersonRiskValue } = {
+export const RiskProfileCurrent: { [key: string]: PersonRiskValue } = {
   average: {
-    label: i18n.t('data.person.average'),
+    label: i18n.t('data.person.notSure_short'),
     personalMultiplier: 1.0,
     ...noContacts,
   },
-  livingAlone: {
-    label: i18n.t('data.person.livingAlone'),
-    personalMultiplier: livingAloneMult,
-    ...noContacts,
-  },
-
   livingWithPartner: {
-    label: i18n.t('data.person.livingWithPartner'),
```

Comment:
(This isn't the right spot for this comment, but I'm adding it here because there's no right spot): Can you open an issue to help us keep track of re-doing the pre-made scenarios to use some of these new risk profiles. I think it's odd that "Live-in partner who has no indoor interactions besides you" bumps the risk profile selector to "follows a risk budget" ...  Because that person doesn't follow a risk budget. We just don't have a risk profile that's really appropriate for them. 

Now that I think about it: we could use the 1-2 people living along, minimal risk profile, even though that's technically double-counting some risk points, it would be more intuitive for the user and still get them fairly close to what that number should be. (Seems like we're going for "ballpark" estimations with our risk profiles currently.)